### ISSUE TITLE
Filter out Sera agreements that are not grants 

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/rialto-etl#211]
`agreementType = 'subcontract'` should be filtered out.

@peetucket is this correct?

**Comment by peetucket** [comment]
not sure, checking with Zach.  What's the downside to including (is there a question of how to map into an ontology)?

**Comment by jcoyne** [comment]
@peetucket the downside is it's more data to manage that we have no use for.  It slows everything down more than necessary.

**Comment by justinlittman** [comment]
@peetucket Can you render a verdict on this ticket?

**Comment by peetucket** [comment]
haven't heard back from zach, inclined to say include it for now since that seems safer than potentially excluding what could be useful data

is this configurable if it needs to changed later?

**Comment by mjgiarlo** [comment]
This is not currently configurable, @peetucket

**Comment by justinlittman** [comment]
But it is just code ...

**Comment by peetucket** [comment]
Yeah, just wondering what the pain level of changing is in between work cycles if it turns out to be a bad decision.
